app/src/main/res/values-sr/strings.xml

```diff
@@ -0,0 +1,160 @@
+    <string name="bluetooth_setting_tracking_explanation_title">Kako slanje podataka radi ?</string>
+    <string name="bluetooth_setting_data_explanation_title">Da li su moji podaci sigurni ?</string>
+    <string name="bluetooth_setting_tracking_explanation_text1">"Preko BLE (Bluetooth Low Energy) aplikacija prepoznaje susrete sa drugim korisnicima. Takvi susreti će biti registrovani od strane aplikacije, ukoliko korisnici stoje blizu jedno drugog određeni vremenski period, i obrisani su posle 21 dan. "</string>
+    <string name="bluetooth_setting_tracking_explanation_text2">Susreti sa drugim korisnicima su sačuvani samo na vašem uređaju.</string>
+    <string name="bluetooth_setting_data_explanation_text1">Svaka aplikacija ima svoj dinamički ID koji je nasumično generisan preko «HMAC(SHA256)» i menja se svakih 15 minuta. Na ovaj način svaki korisnik ostaje anoniman.</string>
+    <string name="bluetooth_setting_data_explanation_text2">Bitne karakteristike: - Ne prati se lokacija - Nema analitike niti reklama - Otvorena arhitektura / protokol i specifikacija uključujući SDK za iOS i Android - Interoperabilnost (sopstvena aplikacija, sopstveni Backend) - Open source</string>
+    <string name="reports_title_homescreen">Izveštaji</string>
+    <string name="meldungen_no_meldungen_title">Nema izveštaja</string>
+    <string name="meldungen_no_meldungen_text">Bićete obavešteni u slučaju moguće infekcije</string>
+    <string name="push_deactivated_title">Push notifikacije onemogućene</string>
+    <string name="push_deactivated_text">Dozvoli push notifikacije da bi bio automatski obavešten kada stigne novi izveštaj.</string>
+    <string name="hinweis_title">Beleška</string>
+    <string name="hinweis_text">Osoba koju ste sreli je testirana pozitivno na COVID-19.</string>
+    <string name="button_report_illness">Prijavi bolest</string>
+    <string name="meldungen_meldung_title">Beleška</string>
+    <string name="meldungen_meldung_text">Osoba koju ste sreli je testirana pozitivno na COVID-19. Molim Vas pročitajte instrukcije ovde.</string>
+    <string name="meldungen_meldung_more_button">Nauči više</string>
+    <string name="tracing_active_title">Slanje podataka omogućeno</string>
+    <string name="tracing_active_text">Susreti su sačuvani anonimno na tvom uređaju.</string>
+    <string name="tracing_error_title">Slanje podataka deaktivirano</string>
+    <string name="tracing_error_text">Aktiviraj slanje podataka i Bluetooth tako da susreti mogu biti sačuvani anonimno.</string>
+    <string name="meldungen_additional_info_title">Šta to znači ?</string>
+    <string name="meldungen_additional_info_text1">Da bi zaustavili viral</string>
+    <string name="meldungen_additional_info_text2">Stoga, izloženi kontakti moraju biti obavešteni. Tada se mogu izolovati ranije, pre nego što se razbole. Neće svaki izložen kontakt imati virus. Ali oni koji ga budu imali ga neće preneti dalje. Cilj je biti jedan korak ispred.</string>
```

Comment:
Stoga, izloženi kontakti moraju biti obavešteni. Tada se mogu izolovati ranije i to pre nego što se razbole. Neće svaki izložen kontakt imati virus, ali oni koji ga budu imali, neće ga prenostit dalje. Cilj je da budemo jedan korak ispred virusa.